configs/jsactions/rollup.config.js

```diff
@@ -45,45 +45,44 @@ export default async args => {
                 i === 0 ? clear({ targets: [outDir] }) : null,
                 !isWeb
                     ? collectDependencies({
-                          onlyNative: false,
-                          outputDir: outDir,
-                          widgetName: fileOutput
-                      })
+                        onlyNative: false,
+                        outputDir: outDir,
+                        widgetName: fileOutput
+                    })
                     : null,
                 nodeResolvePlugin,
                 typescriptPlugin,
                 bigJsImportReplacer(),
-                i === files.length - 1
-                    ? command([
-                          async () => {
-                              if (!isWeb) {
-                                  await Promise.all(
-                                      clientDependencies.map(async dependency => {
-                                          await copyJsModule(
-                                              dirname(require.resolve(`${dependency}/package.json`)),
-                                              join(join(outDir, "node_modules"), dependency)
-                                          );
-                                      })
-                                  );
-                              }
+                command([
```

Comment:
It's not obvious to me why this has changed

Reply:
The way this rollup was configured is that it compiled all files for every file in the directory (iterating over `files` and compiling all files in each iteration..). I updated this logic to just compile the file which is being iterated. This in turn reduces build time and allowed us to just watch one file.